Unable to use chrome for testing (not chromium)

### Version

master

### File / Feature

`npm run test_chromium`

### Expected behaviour

I expect the plugin to build and then start chrome instead of chromium.

### Actual behaviour

After building the plugin the script stops.

### Stacktrace / Error message

```
sh: 1: chromium: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! click-to-dial@2.0.12 test_chromium: `gulp build --target chrome;chromium --user-data-dir=/tmp/click-to-dial --load-extension=./build/chrome --no-first-run`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the click-to-dial@2.0.12 test_chromium script.
```

### Other info

There is a package called `chrome-launcher` to e.g. detect various chrome/chromium installations in PATH. It also supports [chromium on linux](https://github.com/GoogleChrome/lighthouse/commit/1c11021a41334779eed4e80d0d412266ecc1446c).

--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "test_chrome": "gulp build --target chrome;chrome --user-data-dir=/tmp/click-to-dial --load-extension=./build/chrome --no-first-run",
     "test_chromium": "gulp build --target chrome;chromium --user-data-dir=/tmp/click-to-dial --load-extension=./build/chrome --no-first-run",
     "test_electron": "gulp build --target electron;electron --js-flags='--harmony-async-await' build/electron/electron_main.js",
     "test_firefox": "gulp build --target firefox;web-ext run --verbose --source-dir build/firefox",

